types/dot-object/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Dot-Object v1.5
+// Type definitions for Dot-Object v1.7.1
```

Comment:
This should just be `// Type definitions for Dot-Object 1.7`.
Could you remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that might come up? That would have prevented you from making this change.
You're welcome to remove any of the other such lines in that file and fix any errors that might be hidden, but you certainly don't have to. Thanks!